taskana-adapter-camunda-outbox-rest-spring-boot/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>taskana-adapter-camunda-outbox-rest-spring-boot</artifactId>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
```

Comment:
If spring-boot-parent is used as parent, maven installs the lib into 
.m2\repository\org\springframework\boot\taskana-adapter-camunda-outbox-rest-spring-boot\2.0.3.RELEASE\taskana-adapter-camunda-outbox-rest-spring-boot-2.0.3.RELEASE.pom

And from there, it is not found.
We need to set the parent to taskana-adapter-parent